src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -144,25 +157,38 @@ export class LargeArtistHeader extends Component<Props> {
                   </Serif>
                 </H2>
                 <Spacer mr={2} />
-                {props.artist.counts.follows > 50 && (
-                  <Serif size="3">
-                    {props.artist.counts.follows.toLocaleString()} followers
-                  </Serif>
-                )}
               </Flex>
             </Box>
-            <FollowArtistButton
-              useDeprecatedButtonStyle={false}
-              artist={props.artist}
-              user={user}
-              onOpenAuthModal={() =>
-                handleOpenAuth(props.mediator, props.artist)
-              }
-            >
-              Follow
-            </FollowArtistButton>
+            <Flex justifyContent="space-between">
+              {props.artist.counts.follows > 50 && (
+                <Flex flexDirection="column">
+                  <Sans size="4" weight="medium">
+                    {props.artist.counts.follows.toLocaleString()}
+                  </Sans>
+                  <Sans size="2" color="black60" weight="medium">
+                    Followers
+                  </Sans>
+                </Flex>
+              )}
+              <Spacer mr={3} />
+              <FollowArtistButton
+                useDeprecatedButtonStyle={false}
+                artist={props.artist}
+                user={user}
+                onOpenAuthModal={() =>
+                  handleOpenAuth(props.mediator, props.artist)
+                }
+              >
+                Follow
+              </FollowArtistButton>
+            </Flex>
           </Flex>
         </Box>
+        <Spacer mb={1} />
+        <Flex flexDirection="row">
```

Comment:
Does this create extra/unwanted spacing if there is no auction highlight or representation status?

Reply:
Idk why it doesn't say `outdated` but I removed this spacer